src/Controller/Admin/QueueController.php

```diff
@@ -35,7 +36,7 @@ public function initialize(): void {
 	 * Admin center.
 	 * Manage queues from admin backend (without the need to open ssh console window).
 	 *
-	 * @return \Cake\Http\Response|null|void
+	 * @return null|void
```

Comment:
It still could, maybe we shouldnt remove those then?

Reply:
well technically that controller action can't because there is no return anywhere in there. It would be different if we'd enforce `$this->render();` but we don't

Reply:
Documented types these days are never about it what it does so much than what it could.
So it documents that at any point you could start returning it, which is plausible and the whole reason we allow this as range either in types or just documentation :)